mypy/semanal.py

```diff
@@ -1855,8 +1862,14 @@ def add_method(funcname: str,
 
         add_method('_replace', ret=selftype,
                    args=[Argument(var, var.type, EllipsisExpr(), ARG_NAMED_OPT) for var in vars])
+
+        def make_init_arg(var: Var) -> Argument:
+            default = default_items.get(var.name(), None)
```

Comment:
The Python default value could be `None`, but as I understand it mypy would use a value of type `mypy.types.NoneTyp` internally. The previous code also passes `None` to indicate that the `Argument` had no default, so I think this is fine.